html/semantics/scripting-1/the-script-element/nomodule-set-on-async-classic-script.html

```diff
@@ -36,7 +30,7 @@
     script.noModule = false;
     document.body.appendChild(script);
 
-    return waitForAsyncScript().then(() => {
+    return waitForLoadEvent().then(() => {
```

Comment:
Gadzooks, moved too fast when copying local changes over to the GitHub repo.  Fixed now!